azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -1058,7 +1058,7 @@ func apiManagementResourceHostnameSchema(schemaName string) map[string]*schema.S
 		"key_vault_id": {
 			Type:         schema.TypeString,
 			Optional:     true,
-			ValidateFunc: azure.ValidateKeyVaultChildId,
+			ValidateFunc: azure.ValidateKeyVaultChildIdVersionOptional,
```

Comment:
Out of interest - when provisioning this, does the Azure API return the versionless-URI - or the latest version being used?

Reply:
It returns the versionless URI, fortunately.